scipy/stats/_stats_py.py

```diff
@@ -9858,53 +9869,106 @@ def quantile_test(x, *, q=0, p=0.5, alternative='two-sided'):
 
         The following options are available (default is 'two-sided'):
 
-        * 'two-sided': the `p`th-quantile is not `q`
-        * 'less': the `p`th-quantile is less than `q`
-        * 'greater': the `p`th-quantile is greater than `q`
+        * 'two-sided': the :math:`p_{\mathrm{th}}` quantile is not `q`.
+        * 'less': the :math:`p_{\mathrm{th}}` quantile is less than `q`.
+        * 'greater': the :math:`p_{\mathrm{th}}` quantile is greater than `q`.
 
     Returns
     -------
     result : QuantileTestResult
         An object with the following attributes:
 
-        statistic1 : float
-            First test statistic: the proportion of samples in `x` that are
-            less than or equal to the hypothesized quantile `q`
-        statistic2 : float
-            Second test statistic: the proportion of samples in `x` that are
-            strictly less than the hypothesized quantile `q`
+        statistic : float
+            One of two test statistics that may be used in the quantile test.
+            The first test statistic, `T1`, is the proportion of samples in `x`
+            that are less than or equal to the hypothesized quantile `q` The
+            second test statistic, `T2`, is the proportion of samples in `x`
+            that are strictly less than the hypothesized quantile `q`.
+
+            When:
```

Comment:
This shouldn't be bold.
![image](https://github.com/scipy/scipy/assets/6570539/c8c97a9d-b69e-4e82-9c54-952d8e58918e)
I think Sphinx thinks this is the name of a return value. The easiest fix is not to leave a blank line. If that means that we don't get "When" on a line of its own, oh well.